src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -5561,23 +5561,59 @@ void ReportSurfaceHeatBalance(EnergyPlusData &state)
     // This subroutine puts the reporting part of the HBSurface Module in one area.
 
     using SolarShading::ReportSurfaceShading;
-
+    auto &Surface(state.dataSurface->Surface);
     state.dataHeatBalSurf->SumSurfaceHeatEmission = 0.0;
-
+    for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
+        if (Surface(SurfNum).ExtBoundCond == ExternalEnvironment) {
+            state.dataHeatBalSurf->SumSurfaceHeatEmission += state.dataHeatBalSurf->QHeatEmiReport(SurfNum) * state.dataGlobal->TimeStepZoneSec;
+        }
+    }
     ReportSurfaceShading(state);
-    auto &Surface(state.dataSurface->Surface);
 
-    // update inside face radiation reports
-    for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
-        Real64 const surfaceArea(Surface(SurfNum).Area);
+    for (int ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
         // Tuned Replaced by one line form below for speed
         //            QdotRadNetSurfInRep( SurfNum ) = NetLWRadToSurf( SurfNum ) * surfaceArea;
         //            QdotRadNetSurfInRepPerArea( SurfNum ) = NetLWRadToSurf( SurfNum );
-        state.dataHeatBalSurf->QdotRadNetSurfInRep(SurfNum) =
-            (state.dataHeatBalSurf->QdotRadNetSurfInRepPerArea(SurfNum) = state.dataHeatBalSurf->SurfNetLWRadToSurf(SurfNum)) * surfaceArea;
-        state.dataHeatBalSurf->QRadNetSurfInReport(SurfNum) = state.dataHeatBalSurf->QdotRadNetSurfInRep(SurfNum) * state.dataGlobal->TimeStepZoneSec;
-
-        if (Surface(SurfNum).Class != SurfaceClass::Window) { // not a window...
```

Comment:
I also noticed that this test was there and was going to mention it. Glad you caught it.